### PR TITLE
fix problems with dbnd-test-scenarios dependency in tensorflow tests

### PR DESCRIPTION
**Pull request by FHoffmannCode**

### CODE REVIEW

**Review by jonathanshir — APPROVED** [review]

File: plugins/dbnd-tensorflow/setup.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 setuptools.setup(
     name="dbnd-tensorflow",
     package_dir={"": "src"},
-    install_requires=["dbnd==" + version, "tensorflow"],
+    install_requires=["dbnd==" + version, "tensorflow", "h5py==2.10.0"],
```
Comment (evgenyshulman) [inline plugins/dbnd-tensorflow/setup.py:17]: we need it only in tox, we can not enforce the client to use a specific h5py library.

Comment (FHoffmannCode) [reply plugins/dbnd-tensorflow/setup.py:17]: what if your client has non compatible version?

Comment (evgenyshulman) [reply plugins/dbnd-tensorflow/setup.py:17]: it's his problem to fix TensorFlow libraries issue

Comment (FHoffmannCode) [reply plugins/dbnd-tensorflow/setup.py:17]: ok

**Pull request merged**